tool/bump_version.dart

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+import 'dart:io';
+
+void main() async {
+  await bumpVersionInAllPubspecFilesAndAddBlankEntryToChangelogs();
+}
+
+/// When executed, first prompts the user to enter old version, new version, and date.
+/// It then looks for instances of old version in pubspec files and updates them to new version.
+/// It also adds a blank entry to the top of CHANGELOG.md files.
+Future<void> bumpVersionInAllPubspecFilesAndAddBlankEntryToChangelogs() async {
```

Comment:
This might be technically incomplete now that it also updates the top level `amplify.dart`. I think maybe we can make this name slightly less verbose and update the comment to something resembling:
```dart
/// When executed, first prompts the user to enter old version, new version, and date. It then
///   1. looks for instances of old version in pubspec files and updates them to new version.
///   2. replaces the version string in the top-level `amplify.dart` file.
///   3. adds a blank entry to the top of CHANGELOG.md files.
```
That just kind of lists out the changes this function makes?

Reply:
good suggestion, I used this comment and changed to `_bumpVersionInFiles`.